db/migrate/20170725153927_create_data_requests.rb

```diff
@@ -0,0 +1,16 @@
+class CreateDataRequests < ActiveRecord::Migration[5.1]
+  def change
+    create_table :data_requests, id: :uuid do |t|
+      t.integer :user_id
+      t.integer :workflow_id
+      t.string :subgroup
+      t.integer :requested_data
```

Comment:
of these, i think `:requested_data` is the only one we can be confident will never be nil; i might ask for a particular user but not care about the workflow, or vice versa, and i didn't want to require people who were asking for the _default group to explicitly record that fact